.github/workflows/build.yml

```diff
@@ -0,0 +1,10 @@
+name: build
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: docker compose build
```

Comment:
hajp
```suggestion
      - name: Expose GitHub Runtime
        uses: crazy-max/ghaction-github-runtime@v2

      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v2

      - run: |
          docker buildx bake \
            --set "*.cache-from=type=gha" \
            --set "*.cache-to=type=gha,mode=max" \
            --load \
            -f docker-compose.yaml
```

Reply:
Wat das deze to actions do?

Reply:
Första ger access till att använda github actions build cache.
Sen bygger den med buildx magi istället som cachar. Typ samma som man gör med docker build actionen. Men funkar med compose fil

Reply:
Finns nån preview action som funkar med buildx men dno hur bra den är

Reply:
POG!